src/components/atoms/Button.tsx

```diff
@@ -1,21 +1,57 @@
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 import { color, font } from "@/styles";
 
 interface IButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
```

Comment:
```suggestion
interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
```

인터페이스 이름 앞에 인터페이스 약자를 붙이는 건 컨벤션인가요?

Reply:
헉 옛날컨벤션이었는데 컨벤션을 바꾸면서 변경이 안되었네요